homeassistant/components/palazzetti/const.py

```diff
@@ -17,3 +17,53 @@
 FAN_HIGH: Final = "high"
 FAN_AUTO: Final = "auto"
 FAN_MODES: Final = [FAN_SILENT, "1", "2", "3", "4", "5", FAN_HIGH, FAN_AUTO]
+
+STATUSES: Final[dict[str | int | float | None, str]] = {
```

Comment:
```suggestion
STATUS_TO_HA: Final[int, str]] = {
```
just preference to understand that this is a label mapping

Reply:
Out of curiosity, do you think there is value to making this mapping in the library instead of in the integration?

Reply:
personally I'd probably just make it an `IntEnum` in the libary, so I don't have to think/remember what each of that codes is.